ioctl/newcmd/update/update.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package update
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-core/ioctl"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+)
+
+// Multi-language support
+var (
+	shorts = map[config.Language]string{
+		config.English: "Update password for IoTeX account",
+		config.Chinese: "为IoTeX账户更新密码",
+	}
+	uses = map[config.Language]string{
+		config.English: "update [ALIAS|ADDRESS]",
+		config.Chinese: "update [别名|地址]",
+	}
+)
+
+func NewUpdateCmd(c ioctl.Client) *cobra.Command {
+	var versionType string
+	use, _ := c.SelectTranslation(uses)
+	short, _ := c.SelectTranslation(shorts)
```

Comment:
TODO: create function `func MustSelectTranslation(map[config.Language]string) string`

Reply:
> TODO: create function `func MustSelectTranslation(map[config.Language]string) string`

Is it means that I need to create a MustFunction which can recover the panic "failed to pick a translation" to replace `MustSelectTranslation` ?

Reply:
no, you can just leave a todo here.